src/Utility/Inflector.php

```diff
@@ -750,6 +750,10 @@ public static function variable($string)
      */
     public static function slug($string, $replacement = '-')
     {
+        deprecationWarning(
+            'Inflector::slug() is deprecated. ' .
+            'Use Text::getSalt()/setSalt() instead.'
```

Comment:
Shouldn't this be `Text::slug()`?

Reply:
No, you definitely want to get a salt and not a slug.